test/models/cart_test.rb

```diff
@@ -14,4 +14,75 @@ class CartTest < ActiveSupport::TestCase
     assert_equal({ "1" => 2, "2" => 1}, cart.contents)
   end
 
+  test 'can minus_item from cart' do
+    cart = Cart.new({ "1" => 1})
+    cart.add_item(1)
+    cart.add_item(2)
+    cart.minus_item(2)
+
+    assert_equal({"1" =>2, "2"=> 0}, cart.contents)
+  end
+
+  test 'cart can give me the total sum of the items in cart' do
+    drink1 = Item.create(title:"drip", description: 'yum', price: 4, image:'google.com')
+
+    drink2 = Item.create(title:"coffee", description: 'yums', price: 4, image:'google.com')
+
+    cart = Cart.new({})
+
+    cart.add_item(drink1.id)
+    cart.add_item(drink2.id)
+
+
+
+    assert_equal(8, cart.sum)
+  end
+
+  test 'cart can add total quantity ' do
+    drink1 = Item.create(title:"drip", description: 'yum', price: 4, image:'google.com')
+
+    drink2 = Item.create(title:"coffee", description: 'yums', price: 4, image:'google.com')
+
+    cart = Cart.new({})
+
+    cart.add_item(drink1.id)
+    cart.add_item(drink2.id)
+
+    cart.total_quantity
+
+    assert_equal(2, cart.total_quantity)
+  end
+
+  test 'you can see the drinks you added to your cart' do
+    drink1 = Item.create(title:"drip", description: 'yum', price: 4, image:'google.com')
+    drink2 = Item.create(title:"coffee", description: 'yums', price: 4, image:'google.com')
+
+    cart = Cart.new({})
+
+    cart.add_item(drink1.id)
+    cart.add_item(drink2.id)
+
+    expected  = cart.drinks.first.title
+    expected2 = cart.drinks.last.title
+
+    assert_equal("drip", expected)
+    assert_equal("coffee",expected2)
+  end
+
+  test 'cart_item returns items in cart' do
+    drink1 = Item.create(title:"drip", description: 'yum', price: 4, image:'google.com')
+    drink2 = Item.create(title:"coffee", description: 'yums', price: 4, image:'google.com')
```

Comment:
Line is too long. [91/80]<br>Space missing after colon.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.